APMrover2/AP_MotorsUGV.cpp

```diff
@@ -396,6 +423,39 @@ void AP_MotorsUGV::output_regular(bool armed, float steering, float throttle)
     SRV_Channels::set_output_scaled(SRV_Channel::k_steering, steering);
 }
 
+// output for omni style frames
```

Comment:
could we add here the equation/math where the mixing come from ? or a small explanation on how it works ?